guides/common/modules/con_managing-host-packages.adoc

```diff
@@ -0,0 +1,9 @@
+[id="Managing-Host-Packages_{context}"]
+= Managing Host Packages
+
+You can manage the packages of hosts even if the subscriptions are not attached to access repositories.
+Navigate to *Hosts > All Hosts* and select any Host.
```

Comment:
This reads like a procedure.

Reply:
Addressed :+1:

But confused if I should rename the file to `proc` from `con`.

Reply:
Yes, please. I am unsure it it makes sense to still split this into two files: "con_" and "proc_". Maybe someone else can weigh in? @bangelic

Reply:
This reads  similar to proc_removing-host-packages so this should probably be renamed to proc_managing_host_packages.adoc

Reply:
Alright. Thank you @bangelic 
Addressed :+1: